tests/sessions_tests/tests.py

```diff
@@ -660,6 +660,21 @@ def test_httponly_session_cookie(self):
             str(response.cookies[settings.SESSION_COOKIE_NAME])
         )
 
+    @override_settings(SESSION_COOKIE_SAMESITE='Lax')
```

Comment:
This is the default, it is needed? Perhaps another test for a non-default would be useful.